src/main/java/com/entropyteam/entropay/employees/repositories/EmployeeRepository.java

```diff
@@ -5,11 +5,15 @@
 
 import com.entropyteam.entropay.common.BaseRepository;
 import com.entropyteam.entropay.employees.models.Employee;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
 
 public interface EmployeeRepository extends BaseRepository<Employee, UUID> {
 
     List<Employee> findAllByDeletedIsFalse();
 
     List<Employee> findAllByDeletedIsFalseAndActiveIsTrue();
 
+    @Query(value = "SELECT * FROM Employee inner join Contract on Employee.id = Contract.employee_id WHERE Contract.active = true", nativeQuery = true)
+    List<Employee> getEmployeesWithAtLeastAnActiveContract();
```

Comment:
Let's improve this query:
`SELECT e.* FROM Employee AS e inner join Contract AS c ON e.id = c.employee_id WHERE c.active = true and c.deleted = false and e.deleted = false`